pkg/ruler/manager.go

```diff
@@ -171,8 +171,7 @@ func (r *DefaultMultiTenantManager) newManager(ctx context.Context, userID strin
 	reg := prometheus.NewRegistry()
 	r.userManagerMetrics.AddUserRegistry(userID, reg)
 
-	logger := log.With(r.logger, "user", userID)
```

Comment:
Why is per-user logger removed?

Reply:
It's already set as part of the managerFactory.

https://github.com/cortexproject/cortex/blob/5682f6678cd96ce5f3514b4a3484b7d44b22e45a/pkg/ruler/compat.go#L136